docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -29,6 +29,8 @@ IAM credentials, such as an EC2 instance with an attached IAM role.
 No specific permissions or IAM policy is required: an IAM role with no attached
 policies is sufficient.
 No IAM credentials at all are required on the Teleport Auth server.
+The IAM join method is not compatible with the `--insecure-no-tls` flag, and is
```

Comment:
This might work better as an Admonition of type "warning" so it attracts more attention.

Reply:
good call, done